app/models/outstandingCharges/OutstandingChargeModel.scala

```diff
@@ -23,7 +23,10 @@ case class OutstandingChargeModel(chargeName: String,
                                   relevantDueDate: Option[LocalDate],
                                   chargeAmount: BigDecimal,
                                   tieBreaker: Int
-                                 )
+                                 ) {
+
+  def hasRelevantDueDateWithBCDChargeName: Boolean = relevantDueDate.isDefined && chargeName == "BCD"
```

Comment:
Could we find a better name for this method? It seems to be named after it's implementation. What does this method do in domain terms? Could we use that to get a better name?